src/main/java/frc/robot/commands/CancelAll.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.SwerveDrivetrain;
+
+public class CancelAll extends Command {
+  /** Creates a new CancelAll. */
+  SwerveDrivetrain drivetrian;
+  public CancelAll(SwerveDrivetrain drivetrian) {
+    this.drivetrian = drivetrian;
+    addRequirements(drivetrian);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    drivetrian.stopMotor();
```

Comment:
Other motor's from other subsystems should be stopped too, besides just the drivetrain